network/p2p/cache/node_blocklist_wrapper_test.go

```diff
@@ -0,0 +1,169 @@
+package cache_test
```

Comment:
General comment: I think by the term block list in most places, you mean a _black_ list, right?

Reply:
I used the term "blacklist" before but @peterargue requested a terminology change 
> can you switch the terminology from `blacklist` to `blocklist`

My assumption is that "blocklist" is the common terminology for this particular scenario. Maybe @peterargue can add more insights.

Reply:
I suggested it as a less controversial term given this is global open source project. There's a general trend away from `blacklist` and `whitelist` towards `blocklist`/`denylist` and `allowlist`.